### ISSUE TITLE
Almost done setting it up!

### ISSUE THREAD

**Issue by Dokazz** [y-xLeo/discord-token-grabber#5]
Hey, firstly i want to thank you for your awesome work! it seems to be working perfectly, im just stuck on the last step of the installation.
The bot is online, i ran the main.py script (and obviously changed the token and webhook url beforehand), so its connected, but when someone joins the server, nothing happens.
Is it because i placed the webhook in the wrong channel? or the bot in the wrong channel?
basically im stuck here, very close to making this work :)

**Comment by Dokazz** [comment]
I just checked the verify.py script, it says something about a button, do i need to setup a a button with a command that i would put in the channel?
